test/extensions/emotable.ts

```diff
@@ -0,0 +1,89 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import { loadFixture } from '@nomicfoundation/hardhat-network-helpers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { bn } from '../utils';
+import { IERC165, IRMRKMultiAsset, IRMRKEmotable, IOtherInterface } from '../interfaces';
+import { RMRKMultiAssetEmotableMock } from '../../typechain-types';
+
+// --------------- FIXTURES -----------------------
+
+async function multiAssetEmotableFixture() {
+  const factory = await ethers.getContractFactory('RMRKMultiAssetEmotableMock');
+  const token = await factory.deploy('Chunky', 'CHNK');
+  await token.deployed();
+
+  return token;
+}
+
+describe('RMRKMultiAssetEmotableMock', async function () {
+  let token: RMRKMultiAssetEmotableMock;
+  let owner: SignerWithAddress;
+  let addrs: SignerWithAddress[];
+  const tokenId = bn(1);
+  const emoji1 = Buffer.from('😎');
+  const emoji2 = Buffer.from('😁');
+
+  beforeEach(async function () {
+    [owner, ...addrs] = await ethers.getSigners();
+    token = await loadFixture(multiAssetEmotableFixture);
+  });
+
+  it('can support IERC165', async function () {
+    expect(await token.supportsInterface(IERC165)).to.equal(true);
+  });
+
+  it('can support IRMRKMultiAsset', async function () {
+    expect(await token.supportsInterface(IRMRKMultiAsset)).to.equal(true);
+  });
+
+  it('can support IRMRKEmotable', async function () {
+    expect(await token.supportsInterface(IRMRKEmotable)).to.equal(true);
+  });
+
+  it('does not support other interfaces', async function () {
+    expect(await token.supportsInterface(IOtherInterface)).to.equal(false);
+  });
+
+  describe('With minted tokens', async function () {
+    beforeEach(async function () {
+      await token.mint(owner.address, tokenId);
+    });
+
+    it('can emote and undo emote', async function () {
```

Comment:
I think we could split this example into `can emote` and `can undo emote`, since these are two actions that need to be verified 🤔

Reply:
We could.. but meh. Go ahead if you want :sweat_smile:

Reply:
Done 😅